scalafix-sbt/src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -146,8 +146,9 @@ object ScalafixPlugin extends AutoPlugin {
 
   lazy val scalafixScalacSettings: Seq[Def.Setting[_]] = Def.settings(
     scalacOptions ++= {
+      val options = scalafixScalacOptions.value
       if (isSupportedScalaVersion.value) {
-        scalafixScalacOptions.value
+        options
```

Comment:
This is to fix

```
[error] Problem: `scalafixScalacOptions` is inside the if expression of a regular task.
[error]   Regular tasks always evaluate task inside the bodies of if expressions.
[error]
[error] Solution:
[error]   1. If you only want to evaluate it when the if predicate is true or false, use a dynamic task.
[error]   2. Otherwise, make the static evaluation explicit by evaluating `scalafixScalacOptions` outside the if expression.
[error]
[error]         scalafixScalacOptions.value
[error]         ^
[error] one error found
```

Reply:
By the way, ❤️ @jvican for that beautiful error message 👆

Reply:
Thanks @gabro, lovely comment.